backend/coins/eth/account.go

```diff
@@ -334,6 +349,35 @@ func (account *Account) outgoingTransactions(allTxs []*accounts.TransactionData)
 	return transactions, nil
 }
 
+func (account *Account) updateNextNonce() error {
```

Comment:
There is no need to store the nonce in the account struct anymore, you can just return it from this function (and rename the function accordingly).